google/cloud/bigtable/backup.py

```diff
@@ -0,0 +1,392 @@
+        :returns: :class:`~google.cloud.bigtable_admin_v2.types._OperationFuture`
+                  instance, to be used to poll the status of the 'create' request
+        :raises Conflict: if the Backup already exists
+        :raises NotFound: if the Instance owning the Backup does not exist
+        :raises BadRequest: if the `table` or `expire_time` values are invalid,
+                            or `expire_time` is not set
+        """
+        if not self._expire_time:
+            raise ValueError('"expire_time" parameter must be set')
+            # TODO: Consider implementing a method that sets a default value of
+            #  `expire_time`, e.g. 1 week from the creation of the Backup.
+        if not self.table_id:
+            raise ValueError('"table" parameter must be set')
+
+        if cluster_id:
+            self._cluster = cluster_id
+
+        if not self._cluster:
+            raise ValueError('"cluster" parameter must be set')
+
+        backup = table_pb2.Backup(
+            source_table=self.source_table,
+            expire_time=_datetime_to_pb_timestamp(self.expire_time),
+        )
+
+        api = self._instance._client.table_admin_client
+        return api.create_backup(self.parent, self.backup_id, backup)
+
+    def get(self):
+        """Retrieves metadata of a pending or completed Backup.
+
+        :returns: An instance of
+                 :class:`~google.cloud.bigtable_admin_v2.types.Backup`
+
+        :raises google.api_core.exceptions.GoogleAPICallError: If the request
+                failed for any reason.
+        :raises google.api_core.exceptions.RetryError: If the request failed
+                due to a retryable error and retry attempts failed.
+        :raises ValueError: If the parameters are invalid.
+        """
+        api = self._instance._client.table_admin_client
+        try:
+            return api.get_backup(self.name)
+        except NotFound:
+            return None
+
+    def reload(self):
+        """Refreshes the stored backup properties."""
+        backup = self.get()
+        self._source_table = backup.source_table
+        self._expire_time = backup.expire_time
+        self._start_time = backup.start_time
+        self._end_time = backup.end_time
+        self._size_bytes = backup.size_bytes
+        self._state = backup.state
+
+    def exists(self):
+        """Tests whether this Backup exists.
+
+        :rtype: bool
+        :returns: True if the Backup exists, else False.
+        """
+        return self.get() is not None
+
+    def update_expire_time(self, new_expire_time):
+        """Update the expire time of this Backup.
+
+        :type new_expire_time: :class:`datetime.datetime`
+        :param new_expire_time: the new expiration time timestamp
+        """
+        backup_update = table_pb2.Backup(
+            name=self.name, expire_time=_datetime_to_pb_timestamp(new_expire_time),
+        )
+        # update_mask = {"paths": ["expire_time"]}
```

Comment:
Can this line be removed, or is it needed?

Reply:
Absolutely. Sorry, missed it during the cleanup.